### PR TITLE
Internal: Editor V2 - Help package [ED-9734]

### PR DESCRIPTION
**Pull request by StyleShit**

### CODE REVIEW

**Review by rotemee — DISMISSED** [review]

File: core/editor/config-providers/editor-v2-config-provider.php [hunk 1]
```diff
@@ -89,6 +90,8 @@ public function get_client_settings() {
 			'settings' => [
 				'urls' => [
 					'admin' => admin_url(),
+					// TODO: Replace with the final one once available.
```
Comment (Nevoss) [inline core/editor/config-providers/editor-v2-config-provider.php:93]: Ask for the real URL and do not merge it without it.

**Review by Nevoss — CHANGES_REQUESTED** [review]

File: packages/packages/editor/src/contexts/__tests__/settings-context.test.tsx [hunk 2]
```diff
@@ -7,6 +7,7 @@ describe( '@elementor/editor - useSettings()', () => {
 		const settings = {
 			urls: {
 				admin: 'https://localhost/wp-admin/',
+				help: 'https://elementor.com/help',
```
Comment (Nevoss) [inline packages/packages/editor/src/contexts/__tests__/settings-context.test.tsx:10]: Change to some fake url

File: packages/packages/help/package.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (Nevoss) [inline packages/packages/help/package.json:1]: Do we need separated package for `help`? not sure

File: packages/packages/top-bar/src/types.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+type RegisterLink = typeof registerLink;
+type RegisterToggleAction = typeof registerToggleAction;
+
+type ExtractProps<
+	T extends RegisterAction | RegisterLink | RegisterToggleAction
+	> = NonNullable<Parameters<T>[1]['props']>;
+
+export type ActionProps = ExtractProps<RegisterAction>;
+export type ToggleActionProps = ExtractProps<RegisterToggleAction>;
+export type LinkProps = ExtractProps<RegisterLink>;
```
Comment (Nevoss) [inline packages/packages/top-bar/src/types.ts:13]: The props are the component props. so you can use them instead maybe?

**Review by Nevoss — CHANGES_REQUESTED** [review]

File: packages/packages/top-bar/src/types.ts (hunk 4 above)
Comment (StyleShit) [reply packages/packages/top-bar/src/types.ts:13]: Done 😅

File: packages/packages/editor/src/contexts/__tests__/settings-context.test.tsx (hunk 2 above)
Comment (StyleShit) [reply packages/packages/editor/src/contexts/__tests__/settings-context.test.tsx:10]: Done

File: core/editor/config-providers/editor-v2-config-provider.php (hunk 1 above)
Comment (StyleShit) [reply core/editor/config-providers/editor-v2-config-provider.php:95]: Done

File: packages/packages/help/package.json (hunk 3 above)
Comment (StyleShit) [reply packages/packages/help/package.json:1]: NO

**Review by Nevoss — APPROVED** [review]

**Pull request merged**